js/rebase-client/README.md

```diff
@@ -1,137 +1,54 @@
 # Rebase Client
 
-This Rust-to-WASM compiled library is a thin wrapper around the client portion of the [Rebase Witness SDK](https://github.com/spruceid/rebase/tree/main/rust/rebase_witness_sdk). It makes several design choices that make usage in WASM easier. 
+This Rust-to-WASM compiled library is a thin wrapper around the client portion of the [Rebase Witness SDK](../../rust/rebase_witness_sdk). It makes several design choices that make usage in WASM easier. An example of this client in use is [here](../../demo/dapp/).
 
-## Install
-```
-$ npm i @rebase-xyz/rebase-client
-```
-
-## Common Usage:
-```javascript
-import { Client } from "@rebase-xyz/rebase-client";
-
-// const witnessUrl = ...
+Unfortunately, documentation for this project has not yet been written, but please see the [demo front-end project for a working example](../../demo/dapp/).
```

Comment:
Documentation for the Rebase Client is coming soon. For now, please see the [demo front-end project for a working example](../../demo/dapp/).